packages/cloud_firestore/cloud_firestore/lib/src/firestore.dart

```diff
@@ -302,6 +302,24 @@ class FirebaseFirestore extends FirebasePluginPlatform {
     return _delegate.waitForPendingWrites();
   }
 
+  /// Configures indexing for local query execution. Any previous index configuration is overridden.
+  ///
+  /// The index entries themselves are created asynchronously. You can continue to use queries that
+  /// require indexing even if the indices are not yet available. Query execution will automatically
+  /// start using the index once the index entries have been written.
+  Future<void> setIndexConfiguration({
```

Comment:
Should we use the experimental flag here?

Reply:
Good point, I think we should 👌